### PR TITLE
Start 0.19.3

### PR DESCRIPTION
**Pull request by richardjgowers**
Merge once #2133 is done

### CODE REVIEW

File: package/CHANGELOG [hunk 1]
```diff
@@ -13,7 +13,15 @@ The rules for this file:
...
-mm/dd/18 orbeckst, PicoCentauri, richardjgowers
+mm/dd/yy
+  * 0.19.2
+
+Fixes
+
+Deprecations
+
+
+11/05/18 orbeckst, PicoCentauri, richardjgowers, zemanj
```
Comment (orbeckst) [inline package/CHANGELOG]: Shouldn't this have something about Windows conda package? We can say experimental...

**Comment by orbeckst** [comment]
I see a 🍺 coming your way, @richardjgowers ...

File: package/CHANGELOG (hunk 1 above)
Comment (richardjgowers) [reply package/CHANGELOG]: Ah I guess... It's not even experimental tbh, all tests pass

**Comment by richardjgowers** [comment]
Ah I think I lost the beer - I forgot to update the classifiers in setup.py, so pypi doesn't list it as a windows package..

**Comment by orbeckst** [comment]
Then make a 0.19.2. Who cares about the numbers.

**Pull request merged**